pkg/xds/envoy/listeners/listener_builder.go

```diff
@@ -47,6 +47,9 @@ func NewOutboundListenerBuilder(
 	port uint32,
 	protocol core_xds.SocketAddressProtocol,
 ) *ListenerBuilder {
+	if address == "" {
+		address = "127.0.0.1"
```

Comment:
where is this used? I wonder about IPV6

Reply:
This is just the logic we have here https://github.com/kumahq/kuma/blob/master/api/mesh/v1alpha1/dataplane_helpers.go#L239. I can open an issue to figure this out

Reply:
I see, let's leave it as is if this results in the same behavior